src/xunit.analyzers/Descriptors.cs

```diff
@@ -142,6 +142,12 @@ internal static class Descriptors
             "This includes any combination of static and instance methods declared with any visibility in the same class or across a " +
             "class hierarchy. Rename one of the methods.");
 
+        internal static DiagnosticDescriptor X1025_InlineDataShouldBeUniqueWithinTheory { get; } = new DiagnosticDescriptor("xUnit1025",
+            "InlineData should be unique within the Theory it belongs to",
+            "Theory method '{0}' on test class '{1}' has InlineData that duplicates the one declared at '{2}'.",
```

Comment:
Is there some way that these InlineData attributes can be split amongst multiple files? If not, I'd get rid of the logic to print the file name/position and simplify the tests. Typically a collection of inline attributes is easy to scan to find where the duplicate is.